runtime/default.policy

```diff
@@ -31,6 +54,43 @@ macros:
 # Rules determine what the Datadog security-agent will monitor.#
 ################################################################
 rules:
+## Process Activity Rules ##
+  - id: potential_web_shell
+    description: Potential webshell execution
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in ["/usr/bin/curl", "/usr/bin/wget"] || exec.filename in SHELL_UTILS) && process.ancestors.name in WEBAPP_PROCESSES || process.ancestors.name =~ "php*"
+  
+  - id: suspicious_database_process
+    description: Shell or shell utilites spawned by a database process
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in ["/usr/bin/curl", "/usr/bin/wget"] || exec.filename in SHELL_UTILS) && process.ancestors.name in DATABASE_PROCESSES || process.ancestors.name =~ "postgres*"
+  
+  - id: suspicious_java_process
+    description: Shell or shell utilites spawned by a java process
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in ["/usr/bin/curl", "/usr/bin/wget"] || exec.filename in SHELL_UTILS) && process.ancestors.name == "java"
+
+  - id: passwd_process_detected
+    description: passwd process detected
+    expression: >-
+      exec.filename == "/usr/bin/passwd"
+
+  - id: container_management_in_container
+    description: Docker client tool executed inside container
+    expression: >-
+      (exec.filename in CONTAINER_PROCESSES) && container.id != ""
+
+  - id: package_management_in_container
+    description: Package management detected in container
+    expression: >-
+      (exec.filename in PKG_MAN) && container.id != ""
+
+  - id: net_util_in_container
```

Comment:
I am not convinced that we should trigger this rule using the current list of binaries.

Reply:
@guedou are you worried about false positives? Do these utilities commonly get run in containers in your experience? Would love to fine-tune this if you have some feedback

Reply:
Yes, dig & nslookup are useful to debug issues. We might want to list 'dual use' tools such as netcat. That is useful for debugging and also really appreciated to bind shells.